roles/openshift_cfme/tasks/main.yml

```diff
@@ -0,0 +1,176 @@
+# PASSED IN ROLE VARIABLES:
+#
+# - template_dir - A remote temp dir to template the PVs in
+
+---
+- name: What are the groups?
+  debug:
+    var: groups.keys()
+
+- name: Where is my template dir
+  debug:
+    var: template_dir
+
+######################################################################
+- name: Ensure the CFME user exists
+  oc_user:
+    state: present
+    username: "{{ openshift_cfme_user }}"
+
+- name: Ensure the CFME namespace exists with CFME user as admin
+  oc_project:
+    state: present
+    name: "{{ openshift_cfme_project }}"
+    display_name: "{{ openshift_cfme_project_description }}"
+    admin: "{{ openshift_cfme_user }}"
+
+- name: Ensure the CFME namespace service account is privileged
+  oc_adm_policy_user:
+    namespace: "{{ openshift_cfme_project }}"
+    user: "{{ openshift_cfme_service_account }}"
+    resource_kind: scc
+    resource_name: privileged
+    state: present
+
+######################################################################
+- name: Ensure bulk image import limit is tuned
+  yedit:
+    src: /etc/origin/master/master-config.yaml
+    key: 'imagePolicyConfig#maxImagesBulkImportedPerRepository'
+    value: "{{ openshift_cfme_maxImagesBulkImportedPerRepository | int() }}"
+    state: present
+    backup: True
+  register: master_config_updated
+
+# TODO: Figure out how to link to the right handlers rather than
```

Comment:
The handler consolidation work is in progress so I would imagine the [master handlers](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_master/handlers/main.yml) will have to duplicated within this role until `openshift_handlers(?)` is available.